spec/book_spec.rb

```diff
@@ -1,8 +1,16 @@
 require 'rails_helper'
+require 'database_cleaner'
+
+DatabaseCleaner.strategy = :truncation
```

Comment:
For some tips, I advice you to look at the configuration on the [rails-bootstrap](https://github.com/Wolox/rails-bootstrap/blob/master/spec/support/spec_helper.rb)